layer_num = 0
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6940, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6976, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.6926, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.7001, Validation Accuracy: 0.4000
Epoch [5/20], Loss: 0.6931, Validation Accuracy: 0.4000
Epoch [6/20], Loss: 0.6961, Validation Accuracy: 0.4000
Epoch [7/20], Loss: 0.6961, Validation Accuracy: 0.4000
Epoch [8/20], Loss: 0.6911, Validation Accuracy: 0.4000
Epoch [9/20], Loss: 0.6919, Validation Accuracy: 0.4000
Epoch [10/20], Loss: 0.6954, Validation Accuracy: 0.6000
Epoch [11/20], Loss: 0.6956, Validation Accuracy: 0.4000
Epoch [12/20], Loss: 0.6922, Validation Accuracy: 0.4000
Epoch [13/20], Loss: 0.6995, Validation Accuracy: 0.4000
Epoch [14/20], Loss: 0.6930, Validation Accuracy: 0.4000
Epoch [15/20], Loss: 0.6939, Validation Accuracy: 0.4000
Epoch [16/20], Loss: 0.6850, Validation Accuracy: 0.4000
Epoch [17/20], Loss: 0.6928, Validation Accuracy: 0.4000
Epoch [18/20], Loss: 0.6960, Validation Accuracy: 0.4000
Epoch [19/20], Loss: 0.6970, Validation Accuracy: 0.4000
Epoch [20/20], Loss: 0.6880, Validation Accuracy: 0.4000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.5, 'zh-gpt-4': 0.5}
layer_num = 1
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6963, Validation Accuracy: 0.7500
Epoch [2/20], Loss: 0.6833, Validation Accuracy: 0.5500
Epoch [3/20], Loss: 0.6714, Validation Accuracy: 0.5500
Epoch [4/20], Loss: 0.6980, Validation Accuracy: 0.5500
Epoch [5/20], Loss: 0.6789, Validation Accuracy: 0.5500
Epoch [6/20], Loss: 0.5957, Validation Accuracy: 0.6500
Epoch [7/20], Loss: 0.5166, Validation Accuracy: 0.7000
Epoch [8/20], Loss: 0.4776, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.4648, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.3880, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3085, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2705, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.2476, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.1873, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1703, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.1672, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.1161, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.0992, Validation Accuracy: 0.8500
Epoch [19/20], Loss: 0.1399, Validation Accuracy: 0.8500
Epoch [20/20], Loss: 0.1060, Validation Accuracy: 0.8500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9464000000000001, 'zh-gpt-4': 0.916}
layer_num = 2
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6868, Validation Accuracy: 0.6500
Epoch [2/20], Loss: 0.6739, Validation Accuracy: 0.6500
Epoch [3/20], Loss: 0.6177, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.5606, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.4938, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.4552, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.3760, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.2884, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.1779, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.1830, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.1638, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.0704, Validation Accuracy: 0.9500
Epoch [13/20], Loss: 0.0490, Validation Accuracy: 0.9500
Epoch [14/20], Loss: 0.0584, Validation Accuracy: 0.9500
Epoch [15/20], Loss: 0.0848, Validation Accuracy: 0.9500
Epoch [16/20], Loss: 0.0377, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0194, Validation Accuracy: 1.0000
Epoch [18/20], Loss: 0.0274, Validation Accuracy: 0.9500
Epoch [19/20], Loss: 0.0322, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0112, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9892, 'zh-gpt-4': 0.9787999999999999}
layer_num = 3
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6760, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6538, Validation Accuracy: 0.6500
Epoch [3/20], Loss: 0.6222, Validation Accuracy: 0.7000
Epoch [4/20], Loss: 0.5908, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.5340, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.4828, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.3803, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.2648, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.2705, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.1479, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.1859, Validation Accuracy: 0.9500
Epoch [12/20], Loss: 0.1417, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.1353, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0976, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0411, Validation Accuracy: 0.9500
Epoch [16/20], Loss: 0.0341, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0788, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0233, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0271, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0180, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9896, 'zh-gpt-4': 0.9708000000000001}
layer_num = 4
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6934, Validation Accuracy: 0.6500
Epoch [2/20], Loss: 0.6343, Validation Accuracy: 0.6500
Epoch [3/20], Loss: 0.5836, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.5325, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.4627, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.3697, Validation Accuracy: 0.9000
Epoch [7/20], Loss: 0.3207, Validation Accuracy: 0.9000
Epoch [8/20], Loss: 0.1941, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.1411, Validation Accuracy: 1.0000
Epoch [10/20], Loss: 0.1671, Validation Accuracy: 1.0000
Epoch [11/20], Loss: 0.0937, Validation Accuracy: 1.0000
Epoch [12/20], Loss: 0.1230, Validation Accuracy: 1.0000
Epoch [13/20], Loss: 0.0772, Validation Accuracy: 1.0000
Epoch [14/20], Loss: 0.1155, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0513, Validation Accuracy: 1.0000
Epoch [16/20], Loss: 0.0369, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0136, Validation Accuracy: 1.0000
Epoch [18/20], Loss: 0.0572, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0122, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0348, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9823999999999999, 'zh-gpt-4': 0.958}
layer_num = 5
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6959, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6387, Validation Accuracy: 0.7500
Epoch [3/20], Loss: 0.6088, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.5164, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.3428, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.2712, Validation Accuracy: 0.9000
Epoch [7/20], Loss: 0.1965, Validation Accuracy: 0.9500
Epoch [8/20], Loss: 0.1796, Validation Accuracy: 1.0000
Epoch [9/20], Loss: 0.1189, Validation Accuracy: 1.0000
Epoch [10/20], Loss: 0.1047, Validation Accuracy: 1.0000
Epoch [11/20], Loss: 0.0513, Validation Accuracy: 1.0000
Epoch [12/20], Loss: 0.0481, Validation Accuracy: 1.0000
Epoch [13/20], Loss: 0.0340, Validation Accuracy: 1.0000
Epoch [14/20], Loss: 0.0264, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0163, Validation Accuracy: 1.0000
Epoch [16/20], Loss: 0.0301, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0106, Validation Accuracy: 1.0000
Epoch [18/20], Loss: 0.0162, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0176, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0265, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9835999999999999, 'zh-gpt-4': 0.9628}
layer_num = 6
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6910, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6301, Validation Accuracy: 0.7000
Epoch [3/20], Loss: 0.5456, Validation Accuracy: 0.8000
Epoch [4/20], Loss: 0.4429, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.3481, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.2625, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.1308, Validation Accuracy: 0.9500
Epoch [8/20], Loss: 0.1047, Validation Accuracy: 1.0000
Epoch [9/20], Loss: 0.0680, Validation Accuracy: 1.0000
Epoch [10/20], Loss: 0.0673, Validation Accuracy: 1.0000
Epoch [11/20], Loss: 0.0215, Validation Accuracy: 1.0000
Epoch [12/20], Loss: 0.0240, Validation Accuracy: 1.0000
Epoch [13/20], Loss: 0.0210, Validation Accuracy: 1.0000
Epoch [14/20], Loss: 0.0104, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0099, Validation Accuracy: 1.0000
Epoch [16/20], Loss: 0.0061, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0080, Validation Accuracy: 1.0000
Epoch [18/20], Loss: 0.0155, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0049, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0036, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9835999999999999, 'zh-gpt-4': 0.9704}
layer_num = 7
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6949, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6308, Validation Accuracy: 0.8000
Epoch [3/20], Loss: 0.5122, Validation Accuracy: 0.8500
Epoch [4/20], Loss: 0.3471, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.2435, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.1927, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.0850, Validation Accuracy: 0.9000
Epoch [8/20], Loss: 0.0573, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.0469, Validation Accuracy: 0.9500
Epoch [10/20], Loss: 0.0136, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0095, Validation Accuracy: 0.9500
Epoch [12/20], Loss: 0.0076, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0109, Validation Accuracy: 1.0000
Epoch [14/20], Loss: 0.0071, Validation Accuracy: 0.9500
Epoch [15/20], Loss: 0.0046, Validation Accuracy: 0.9500
Epoch [16/20], Loss: 0.0092, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0048, Validation Accuracy: 0.9500
Epoch [18/20], Loss: 0.0045, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0007, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0004, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9916, 'zh-gpt-4': 0.9768}
layer_num = 8
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6815, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6110, Validation Accuracy: 0.7500
Epoch [3/20], Loss: 0.5589, Validation Accuracy: 0.8000
Epoch [4/20], Loss: 0.3882, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.2826, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.1364, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.1188, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.0972, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.0904, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0656, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0219, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0210, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0399, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0103, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0090, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0077, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0065, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0059, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0032, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0022, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.992, 'zh-gpt-4': 0.9804}
layer_num = 9
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7362, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6506, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.6677, Validation Accuracy: 0.7000
Epoch [4/20], Loss: 0.6741, Validation Accuracy: 0.6000
Epoch [5/20], Loss: 0.6155, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.6712, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.7946, Validation Accuracy: 0.6000
Epoch [8/20], Loss: 0.5527, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.5549, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.5270, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3119, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.3638, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.4747, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.4860, Validation Accuracy: 0.8000
Epoch [15/20], Loss: 0.4312, Validation Accuracy: 0.8000
Epoch [16/20], Loss: 0.3365, Validation Accuracy: 0.8000
Epoch [17/20], Loss: 0.2442, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.2718, Validation Accuracy: 0.8500
Epoch [19/20], Loss: 0.2213, Validation Accuracy: 0.8500
Epoch [20/20], Loss: 0.1616, Validation Accuracy: 0.8000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.99, 'zh-gpt-4': 0.9736}
layer_num = 10
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7166, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6984, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.6872, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.8266, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.5237, Validation Accuracy: 0.6500
Epoch [6/20], Loss: 0.5698, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.6977, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.4616, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.6557, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.2798, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3827, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.2738, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.2411, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.2123, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.2951, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.1719, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.2394, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.1412, Validation Accuracy: 0.8500
Epoch [19/20], Loss: 0.0767, Validation Accuracy: 0.8500
Epoch [20/20], Loss: 0.0930, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9872000000000001, 'zh-gpt-4': 0.9703999999999999}
layer_num = 11
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6356, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.5069, Validation Accuracy: 0.7000
Epoch [3/20], Loss: 0.7778, Validation Accuracy: 0.8000
Epoch [4/20], Loss: 0.7072, Validation Accuracy: 0.6500
Epoch [5/20], Loss: 0.7029, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.6438, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.5023, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.4890, Validation Accuracy: 0.7500
Epoch [9/20], Loss: 0.3702, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.5160, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3230, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.4531, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.1369, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.1732, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.0939, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.1152, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.2208, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0719, Validation Accuracy: 0.9500
Epoch [19/20], Loss: 0.0508, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0365, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9904, 'zh-gpt-4': 0.976}
layer_num = 12
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6899, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.7865, Validation Accuracy: 0.8000
Epoch [3/20], Loss: 0.7441, Validation Accuracy: 0.8000
Epoch [4/20], Loss: 0.5775, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.5198, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.5049, Validation Accuracy: 0.7500
Epoch [7/20], Loss: 0.7382, Validation Accuracy: 0.8000
Epoch [8/20], Loss: 0.5457, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.6093, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.3798, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.2567, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2902, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.3408, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.2296, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.1064, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0867, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.1377, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0979, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0606, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0288, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9904, 'zh-gpt-4': 0.9780000000000001}
layer_num = 13
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7989, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6967, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.7147, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.7377, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.6871, Validation Accuracy: 0.6500
Epoch [6/20], Loss: 0.5880, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.4744, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.4928, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.3527, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.2318, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3060, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.3141, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.1669, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.1522, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1633, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.0929, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.1160, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.1078, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0619, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0655, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9916, 'zh-gpt-4': 0.9756}
layer_num = 14
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8239, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.5885, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.8091, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.5782, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.6633, Validation Accuracy: 0.7000
Epoch [6/20], Loss: 0.6470, Validation Accuracy: 0.7500
Epoch [7/20], Loss: 0.4577, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.4603, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.4431, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.3870, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.4085, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.2617, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.3082, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.1194, Validation Accuracy: 0.8000
Epoch [15/20], Loss: 0.1279, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.1805, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0912, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0800, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0746, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0473, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9908, 'zh-gpt-4': 0.9752000000000001}
layer_num = 15
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6967, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.7915, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.8113, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.5994, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.7945, Validation Accuracy: 0.6500
Epoch [6/20], Loss: 0.4589, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.5449, Validation Accuracy: 0.7500
Epoch [8/20], Loss: 0.5983, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.4141, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.3465, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.4104, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2781, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.2200, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.2468, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1372, Validation Accuracy: 0.8000
Epoch [16/20], Loss: 0.1508, Validation Accuracy: 0.8000
Epoch [17/20], Loss: 0.1988, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.2346, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0800, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0482, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9908, 'zh-gpt-4': 0.9728}
layer_num = 16
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8184, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6318, Validation Accuracy: 0.8500
Epoch [3/20], Loss: 0.5798, Validation Accuracy: 0.4000
Epoch [4/20], Loss: 0.8097, Validation Accuracy: 0.6500
Epoch [5/20], Loss: 0.6432, Validation Accuracy: 0.7000
Epoch [6/20], Loss: 0.6409, Validation Accuracy: 0.6500
Epoch [7/20], Loss: 0.5574, Validation Accuracy: 0.7000
Epoch [8/20], Loss: 0.6301, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.4911, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.5399, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.4212, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.3391, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.2054, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.2758, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1730, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.1510, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.1478, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.2055, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.1716, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0652, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9884, 'zh-gpt-4': 0.9692000000000001}
layer_num = 17
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8594, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6451, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.7196, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.6059, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.7779, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.7077, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.5554, Validation Accuracy: 0.8000
Epoch [8/20], Loss: 0.6587, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.4424, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.5597, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.4858, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.4358, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.4287, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.3030, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.4816, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.2986, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.2130, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.1200, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.1287, Validation Accuracy: 0.8500
Epoch [20/20], Loss: 0.2431, Validation Accuracy: 0.8500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9884000000000001, 'zh-gpt-4': 0.9704}
layer_num = 18
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8112, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.6784, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.6560, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.6941, Validation Accuracy: 0.6000
Epoch [5/20], Loss: 0.5224, Validation Accuracy: 0.6000
Epoch [6/20], Loss: 0.5649, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.7059, Validation Accuracy: 0.7000
Epoch [8/20], Loss: 0.4616, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.4375, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.3842, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.3412, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.3017, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.3391, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.2487, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1537, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.2741, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0890, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.1102, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0706, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.2132, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9944, 'zh-gpt-4': 0.9752}
layer_num = 19
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7770, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.5491, Validation Accuracy: 0.6500
Epoch [3/20], Loss: 0.7090, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.6015, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.6515, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.5819, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.5382, Validation Accuracy: 0.8000
Epoch [8/20], Loss: 0.4040, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.5170, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.5271, Validation Accuracy: 0.8000
Epoch [11/20], Loss: 0.5322, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.4515, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.1999, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.3333, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.2117, Validation Accuracy: 0.8000
Epoch [16/20], Loss: 0.1039, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.1964, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.2142, Validation Accuracy: 0.8500
Epoch [19/20], Loss: 0.1409, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.1004, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.994, 'zh-gpt-4': 0.9763999999999999}
layer_num = 20
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8253, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.5912, Validation Accuracy: 0.4000
Epoch [3/20], Loss: 0.4722, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.6378, Validation Accuracy: 0.7000
Epoch [5/20], Loss: 0.4272, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.4527, Validation Accuracy: 0.7000
Epoch [7/20], Loss: 0.5241, Validation Accuracy: 0.7500
Epoch [8/20], Loss: 0.3311, Validation Accuracy: 0.7500
Epoch [9/20], Loss: 0.4874, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.4675, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.3350, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2088, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.2295, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.2162, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.0964, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.2437, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.2014, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0744, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0923, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0361, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.996, 'zh-gpt-4': 0.9848}
layer_num = 21
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.5839, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.8070, Validation Accuracy: 0.5000
Epoch [3/20], Loss: 0.7710, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.6650, Validation Accuracy: 0.7000
Epoch [5/20], Loss: 0.4541, Validation Accuracy: 0.7000
Epoch [6/20], Loss: 0.4595, Validation Accuracy: 0.7000
Epoch [7/20], Loss: 0.4944, Validation Accuracy: 0.7000
Epoch [8/20], Loss: 0.5150, Validation Accuracy: 0.7500
Epoch [9/20], Loss: 0.3800, Validation Accuracy: 0.7500
Epoch [10/20], Loss: 0.4055, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.2460, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2847, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.2126, Validation Accuracy: 0.8000
Epoch [14/20], Loss: 0.1775, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.3388, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.3088, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.1425, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0655, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0665, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0399, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9967999999999999, 'zh-gpt-4': 0.9836}
layer_num = 22
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7566, Validation Accuracy: 0.5500
Epoch [2/20], Loss: 0.6781, Validation Accuracy: 0.5500
Epoch [3/20], Loss: 0.5455, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.6322, Validation Accuracy: 0.6000
Epoch [5/20], Loss: 0.4831, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.4633, Validation Accuracy: 0.7500
Epoch [7/20], Loss: 0.4811, Validation Accuracy: 0.7500
Epoch [8/20], Loss: 0.5918, Validation Accuracy: 0.7500
Epoch [9/20], Loss: 0.3053, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.2700, Validation Accuracy: 0.7500
Epoch [11/20], Loss: 0.2193, Validation Accuracy: 0.8000
Epoch [12/20], Loss: 0.2458, Validation Accuracy: 0.8000
Epoch [13/20], Loss: 0.1426, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.0892, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.1583, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.1593, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0503, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0253, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0905, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0240, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9904000000000001, 'zh-gpt-4': 0.9696}
layer_num = 23
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7930, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.7077, Validation Accuracy: 0.7000
Epoch [3/20], Loss: 0.6320, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.6485, Validation Accuracy: 0.6000
Epoch [5/20], Loss: 0.7963, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.4330, Validation Accuracy: 0.7500
Epoch [7/20], Loss: 0.4440, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.3181, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.4725, Validation Accuracy: 0.8000
Epoch [10/20], Loss: 0.2679, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.1447, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.1194, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.2119, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.0558, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0417, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0364, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0420, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0203, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0162, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0307, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9884, 'zh-gpt-4': 0.9683999999999999}
layer_num = 24
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.5879, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.7234, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.5513, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.7203, Validation Accuracy: 0.6000
Epoch [5/20], Loss: 0.4506, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.3063, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.2665, Validation Accuracy: 0.7500
Epoch [8/20], Loss: 0.2719, Validation Accuracy: 0.7500
Epoch [9/20], Loss: 0.2836, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.1243, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.0970, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.0569, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.0624, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.0465, Validation Accuracy: 0.8500
Epoch [15/20], Loss: 0.0315, Validation Accuracy: 0.8500
Epoch [16/20], Loss: 0.0602, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.0235, Validation Accuracy: 0.8500
Epoch [18/20], Loss: 0.0349, Validation Accuracy: 0.9500
Epoch [19/20], Loss: 0.0369, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0156, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9868, 'zh-gpt-4': 0.964}
layer_num = 25
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.6170, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.5769, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.3510, Validation Accuracy: 0.5500
Epoch [4/20], Loss: 0.8087, Validation Accuracy: 0.7500
Epoch [5/20], Loss: 0.5225, Validation Accuracy: 0.7500
Epoch [6/20], Loss: 0.3908, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.3926, Validation Accuracy: 0.7500
Epoch [8/20], Loss: 0.2027, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.1620, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.3104, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.1221, Validation Accuracy: 0.8500
Epoch [12/20], Loss: 0.0830, Validation Accuracy: 0.8500
Epoch [13/20], Loss: 0.1288, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.0793, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0410, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0631, Validation Accuracy: 0.8500
Epoch [17/20], Loss: 0.0455, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0423, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0129, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0023, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9848, 'zh-gpt-4': 0.9628}
layer_num = 26
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8148, Validation Accuracy: 0.4000
Epoch [2/20], Loss: 0.5833, Validation Accuracy: 0.7000
Epoch [3/20], Loss: 0.5322, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.5379, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.2744, Validation Accuracy: 0.6500
Epoch [6/20], Loss: 0.3605, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.3718, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.2033, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.1092, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.0397, Validation Accuracy: 0.9500
Epoch [11/20], Loss: 0.0636, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0205, Validation Accuracy: 0.9500
Epoch [13/20], Loss: 0.0527, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0149, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0500, Validation Accuracy: 0.9500
Epoch [16/20], Loss: 0.0108, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0322, Validation Accuracy: 0.9500
Epoch [18/20], Loss: 0.0035, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0030, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0028, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9783999999999999, 'zh-gpt-4': 0.9528}
layer_num = 27
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7035, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.5858, Validation Accuracy: 0.7500
Epoch [3/20], Loss: 0.4899, Validation Accuracy: 0.6000
Epoch [4/20], Loss: 0.4765, Validation Accuracy: 0.6500
Epoch [5/20], Loss: 0.4736, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.2720, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.2542, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.1692, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.1325, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0359, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0663, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0414, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0132, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0213, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0110, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0161, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0060, Validation Accuracy: 1.0000
Epoch [18/20], Loss: 0.0048, Validation Accuracy: 1.0000
Epoch [19/20], Loss: 0.0044, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0124, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9736, 'zh-gpt-4': 0.9491999999999999}
layer_num = 28
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7236, Validation Accuracy: 0.4500
Epoch [2/20], Loss: 0.6467, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.5494, Validation Accuracy: 0.7000
Epoch [4/20], Loss: 0.5741, Validation Accuracy: 0.7500
Epoch [5/20], Loss: 0.2346, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.1414, Validation Accuracy: 0.8000
Epoch [7/20], Loss: 0.1296, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.1260, Validation Accuracy: 0.8000
Epoch [9/20], Loss: 0.1034, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0368, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0323, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0114, Validation Accuracy: 0.9500
Epoch [13/20], Loss: 0.0107, Validation Accuracy: 1.0000
Epoch [14/20], Loss: 0.0161, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0212, Validation Accuracy: 1.0000
Epoch [16/20], Loss: 0.0099, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0021, Validation Accuracy: 0.9500
Epoch [18/20], Loss: 0.0090, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0016, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0017, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.978, 'zh-gpt-4': 0.9516000000000001}
layer_num = 29
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7969, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.7005, Validation Accuracy: 0.6000
Epoch [3/20], Loss: 0.4707, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.3374, Validation Accuracy: 0.7000
Epoch [5/20], Loss: 0.1922, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.1507, Validation Accuracy: 0.7500
Epoch [7/20], Loss: 0.2015, Validation Accuracy: 0.9000
Epoch [8/20], Loss: 0.1095, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.0774, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0311, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0145, Validation Accuracy: 1.0000
Epoch [12/20], Loss: 0.0106, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0141, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0050, Validation Accuracy: 1.0000
Epoch [15/20], Loss: 0.0081, Validation Accuracy: 1.0000
Epoch [16/20], Loss: 0.0016, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0037, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0020, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0081, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0018, Validation Accuracy: 1.0000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9723999999999999, 'zh-gpt-4': 0.9508}
layer_num = 30
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.8000, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.4686, Validation Accuracy: 0.7500
Epoch [3/20], Loss: 0.3703, Validation Accuracy: 0.6500
Epoch [4/20], Loss: 0.3193, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.1622, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.1608, Validation Accuracy: 0.9000
Epoch [7/20], Loss: 0.0734, Validation Accuracy: 0.9000
Epoch [8/20], Loss: 0.0504, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.0198, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0069, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0085, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0251, Validation Accuracy: 0.9500
Epoch [13/20], Loss: 0.0217, Validation Accuracy: 0.9500
Epoch [14/20], Loss: 0.0022, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0038, Validation Accuracy: 0.9500
Epoch [16/20], Loss: 0.0036, Validation Accuracy: 0.9500
Epoch [17/20], Loss: 0.0106, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0043, Validation Accuracy: 0.9500
Epoch [19/20], Loss: 0.0039, Validation Accuracy: 0.9500
Epoch [20/20], Loss: 0.0068, Validation Accuracy: 0.9500
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9604, 'zh-gpt-4': 0.9344}
layer_num = 31
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.7165, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.6026, Validation Accuracy: 0.6500
Epoch [3/20], Loss: 0.3982, Validation Accuracy: 0.7000
Epoch [4/20], Loss: 0.2559, Validation Accuracy: 0.8000
Epoch [5/20], Loss: 0.2316, Validation Accuracy: 0.8000
Epoch [6/20], Loss: 0.1693, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.0415, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.0408, Validation Accuracy: 0.9000
Epoch [9/20], Loss: 0.0166, Validation Accuracy: 0.9000
Epoch [10/20], Loss: 0.0221, Validation Accuracy: 0.9000
Epoch [11/20], Loss: 0.0098, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0039, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0060, Validation Accuracy: 0.9000
Epoch [14/20], Loss: 0.0043, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0037, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0031, Validation Accuracy: 1.0000
Epoch [17/20], Loss: 0.0016, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0022, Validation Accuracy: 0.9000
Epoch [19/20], Loss: 0.0086, Validation Accuracy: 1.0000
Epoch [20/20], Loss: 0.0012, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9588, 'zh-gpt-4': 0.9321999999999999}
layer_num = 32
trying to get zh_train
dataset/processed_multitude/train_valid_data/zh_train.json
trying to get zh_val
dataset/processed_multitude/train_valid_data/zh_val.json
the device being used is cuda
there are 150 train data and 20 validation data.
Epoch [1/20], Loss: 0.9335, Validation Accuracy: 0.6000
Epoch [2/20], Loss: 0.3058, Validation Accuracy: 0.8000
Epoch [3/20], Loss: 0.1760, Validation Accuracy: 0.7500
Epoch [4/20], Loss: 0.0913, Validation Accuracy: 0.8500
Epoch [5/20], Loss: 0.1183, Validation Accuracy: 0.8500
Epoch [6/20], Loss: 0.0209, Validation Accuracy: 0.8500
Epoch [7/20], Loss: 0.0123, Validation Accuracy: 0.8500
Epoch [8/20], Loss: 0.0211, Validation Accuracy: 0.8500
Epoch [9/20], Loss: 0.0127, Validation Accuracy: 0.8500
Epoch [10/20], Loss: 0.0027, Validation Accuracy: 0.8500
Epoch [11/20], Loss: 0.0041, Validation Accuracy: 0.9000
Epoch [12/20], Loss: 0.0038, Validation Accuracy: 0.9000
Epoch [13/20], Loss: 0.0022, Validation Accuracy: 0.8500
Epoch [14/20], Loss: 0.0028, Validation Accuracy: 0.9000
Epoch [15/20], Loss: 0.0056, Validation Accuracy: 0.9000
Epoch [16/20], Loss: 0.0021, Validation Accuracy: 0.9000
Epoch [17/20], Loss: 0.0017, Validation Accuracy: 0.9000
Epoch [18/20], Loss: 0.0018, Validation Accuracy: 0.8500
Epoch [19/20], Loss: 0.0010, Validation Accuracy: 0.9000
Epoch [20/20], Loss: 0.0007, Validation Accuracy: 0.9000
trying to get zh_gpt-3.5-turbo
dataset/processed_multitude/test_data/zh_gpt-3.5-turbo.json
trying to get zh_gpt-4
dataset/processed_multitude/test_data/zh_gpt-4.json
{'zh-gpt-3.5-turbo': 0.9512, 'zh-gpt-4': 0.922}
